its/plugin/src/test/java/com/sonar/it/xml/XmlTest.java

```diff
@@ -51,7 +51,7 @@ public void testBaseMetrics() {
     assertThat(getProjectMeasureAsDouble("comment_lines_density")).isEqualTo(3.1);
     assertThat(getProjectMeasureAsDouble("comment_lines")).isEqualTo(9);
     assertThat(getProjectMeasureAsDouble("files")).isEqualTo(4);
-    assertThat(getProjectMeasureAsDouble("violations")).isEqualTo(19);
+    assertThat(getProjectMeasureAsDouble("violations")).isEqualTo(11);
```

Comment:
We are changing the behavior of the rule, so I think we should also update the ticket to make it explicit how we changed the rule.

Reply:
done